ctfpad/templates/users/edit_advanced.html

```diff
@@ -1,69 +1,89 @@
-{%  extends 'ctfpad/main.html' %}
+{% extends 'ctfpad/main.html' %}
 
 {% block content %}
 
-<br/>
+    <br/>
 
-{% include 'snippets/messages.html' %}
+    {% include 'snippets/messages.html' %}
 
-{{ form.errors }}
-
-<div style="padding-left: 33%;">
+    {% if form.errors %}
+        <div class="row">
+            <div class="col-lg-4 offset-lg-4">
+                <div class="alert alert-danger">
+                    <strong>ERROR</strong>&nbsp;&nbsp;
+                    {{ form.errors }}
+                </div>
+            </div>
+        </div>
+    {% endif %}
```

Comment:
This pattern appears multiple times: it could be better to factorize it into a snippets and only include the snippet.

Reply:
Done :)